spec/helpers/content_helper_spec.rb

```diff
@@ -128,4 +128,21 @@
       end
     end
   end
+
+  describe '#card_thumbnail' do
+    let(:null_resource) { helper.null_resource('example') }
```

Comment:
This is a very confusing spec. Does it give us greater confidence in our code doing the right thing?